Admin/Test-ExchangePropertyPermissions/Get-TokenGroupsGlobalAndUniversal.ps1

```diff
@@ -0,0 +1,39 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function Get-TokenGroupsGlobalAndUniversal {
```

Comment:
Is there a reason why we are creating a new function basically doing the same thing here as `Shared\ActiveDirectoryFunctions\Get-TokenGroupsGlobalAndUniversal.ps1`

Reply:
The reason is that I forgot I had already copied it into Shared. :D